A workaround for error when building for riscv64.

Upstream report: https://bugreports.qt.io/browse/QTBUG-84580

Patch concept: https://bugs.gentoo.org/790689


--- old/qtdeclarative/src/qml/qml.pro
+++ new/qtdeclarative/src/qml/qml.pro
@@ -19,6 +19,7 @@ solaris-cc*:QMAKE_CXXFLAGS_RELEASE -= -O2
 
 # Ensure this gcc optimization is switched off for mips platforms to avoid trouble with JIT.
 gcc:isEqual(QT_ARCH, "mips"): QMAKE_CXXFLAGS += -fno-reorder-blocks
+gcc:isEqual(QT_ARCH, "riscv64"): LIBS += -latomic
 
 DEFINES += QT_NO_FOREACH
 
